enforcer/remote/remenforcer/remEnforcer.go

```diff
@@ -0,0 +1,225 @@
+//Package enforcerLauncher :: This is the implementation of the RPC client
+//It implementes the interface PolicyEnforcer and forwards these requests to the actual enforcer
+package remenforcer
```

Comment:
This package is  `enforcer/remote/remenforcer`
Canit be moved to `enforcer/remote` instead ?